### DISCUSSION TITLE
Google Drive Disk

### DISCUSSION THREAD

**Discussion by Armnum** [iterate-ch/cyberduck#15058] [Q&A]
Hi. I've installed Mountainduck on my Notebook and on my PC, both with Windows10 as OS. While in my notebook (a Dell XPS 16) it works perfectly, on my PC (a HP ProDesk), instead, it shows me regularly the Google Drive Disk (as disk F:), but if I create a new file or folder or if I modify a file or a folder, on Google Drive I can't find any new file or any new folder. Instead, if I do the same activities on my Notebook, every single change is correctly sync and registered also in the Goolge Drive online platform. There is someone that had the same problem and fixed it? Thank you.

**Comment by tatze96** [comment]
Based on your description, it seems that the synchronization process on your HP ProDesk is paused, and the file listing doesn't get refreshed because of it.

The synchronization process can be paused manually but also gets paused by Mountain Duck if the connection to the server is lost or too unstable to transfer files. Files and folders that weren't indexed before the server or network connection dropped won't show up if you work offline.

If the synchronization process was paused by Mountain Duck, it will resume automatically once the connection can be established again.

Please verify that the synchronization process isn't [paused](https://docs.cyberduck.io/mountainduck/connect/sync/#pause-sync) within the Mountain Duck menu if there are items that don't show up.
